components/ChallengeMaterial/__snapshots__/ChallengeMaterial.test.js.snap

```diff
@@ -623,16 +623,18 @@ exports[`Curriculum challenge page Should render first challenge that is not pas
               class="border border-lightgray rounded my-1"
             >
               <div
-                class="ms-3"
+                class="mx-3"
               >
-                <span
-                  wrapper="code_wrapper"
+                <div
+                  class="my-3"
```

Comment:
The content doesn't have consistent vertical spacing and the text is so close to the top border. Adding a top & bottom margin fixes it.